script_library/requirements.txt

```diff
@@ -4,3 +4,5 @@ jmespath==0.9.3
 requests==2.21.0
 ara==0.16.3
 openstacksdk==0.22.0
+python-openstackclient
+python-heatclient
```

Comment:
LGTM. Just wondering why we didnot see this issue earlier in virtualenv case ? Is it something changed recently as I am seeing issue with caasp v3 install case

Reply:
I haven't been able to figure out what changed either, I have no idea why this is just now showing up